.env.example

```
@@ -123,11 +123,14 @@ REDIS_URL=redis://localhost:6379/1
 # This is the domain that your Sure instance will be hosted at. It is used to generate links in emails and other places.
 APP_DOMAIN=
 
-# WebAuthn / passkey MFA configuration
+# WebAuthn / passkey configuration
 # RP ID is usually the registrable domain (example.com), not a full URL.
 # Allowed origins are full HTTPS origins where users access Sure.
 WEBAUTHN_RP_ID=
 WEBAUTHN_ALLOWED_ORIGINS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEBAUTHN_ALLOWED_ORIGINS key should go before the WEBAUTHN_RP_ID key

(UnorderedKey)